Directory.Build.props

```diff
@@ -54,10 +54,22 @@
     <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
-    <!-- TODO: Might be related to: https://github.com/dotnet/project-system/issues/5619 -->
-    <EnableSourceLink Condition="'$(CIBuild)' == 'true'">true</EnableSourceLink>
     <NoWarn>$(NoWarn);NU5125</NoWarn>
 
+    <!-- SourceLink properties -->
+    <!-- https://github.com/dotnet/sourcelink#using-source-link-in-net-projects -->
+    <EnableSourceLink>true</EnableSourceLink>
+    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!-- Embed source files that are not tracked by the source control manager in the PDB -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!-- Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <!-- Marks the assembly be built in a determinstic way. -->
```

Comment:
`determinstic` is misspelled

Reply:
Derp! Missed an `i` in there.

Reply:
My extensions in VSCode somehow all disabled themselves. They all say "Reload Required" beside them. Explains why I didn't see the spelling error. 🤔